src/test/java/uk/gov/pay/api/json/CreateCardPaymentRequestDeserializerTest.java

```diff
@@ -162,7 +170,7 @@ public void deserialize_shouldDeserializeARequestWithMotoEqualsTrueSuccessfully(
         assertThat(paymentRequest.getReference(), is("Some reference"));
         assertThat(paymentRequest.getDescription(), is("Some description"));
         assertThat(paymentRequest.getReturnUrl(), is("https://somewhere.gov.uk/rainbow/1"));
-        assertThat(paymentRequest.getMoto(), is(Optional.of(Boolean.TRUE)));
+        assertThat(paymentRequest.getMoto(), is(Optional.of(Boolean.parseBoolean(value))));
```

Comment:
the test `deserialize_shouldDeserializeARequestWithMotoEqualsFalseSuccessfully` below could be deleted now

Reply:
Oh yes sorry! Done